manifests/01_namespace.yaml

```diff
@@ -10,4 +10,6 @@ metadata:
     capability.openshift.io/name: "marketplace"
   labels:
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: v1.24
```

Comment:
/hold 

We just label it to enforce baseline because the pod requires more permissions.
We do not need to enforce it as restricted because it is done by default

Reply:
all pods running in this NS are running as restricted-v2, so setting PSA enforcement to restricted should be fine.

```
$ oc project openshift-marketplace
Now using project "openshift-marketplace" on server "https://api.bparees.devcluster.openshift.com:6443".

$ oc get pods -o yaml | grep scc
      openshift.io/scc: restricted-v2
      openshift.io/scc: restricted-v2
      openshift.io/scc: restricted-v2
      openshift.io/scc: restricted-v2
      openshift.io/scc: restricted-v2
```

Reply:
So, we get these changes merged. 👍 
/hold cancel

Reply:
/lgtm

Reply:
/hold cancel